examples/lock-app/nrfconnect/main/include/CHIPProjectConfig.h

```diff
@@ -31,4 +31,4 @@
 #define CHIP_DEVICE_CONFIG_USE_TEST_SETUP_PIN_CODE 20202021
 #define CHIP_DEVICE_CONFIG_USE_TEST_SETUP_DISCRIMINATOR 0xF00
 
-#define CHIP_DEVICE_CONFIG_SED_IDLE_INTERVAL 2000_ms32
+#define CHIP_DEVICE_CONFIG_SED_IDLE_INTERVAL 20000_ms32
```

Comment:
In fact, the original 2s is too short for a sleepy device -- the MRP timeout is 5s for a non-sleepy device.

Reply:
This is not a timeout, it's just an interval which determines how often the device wakes up, so it can be anything. I think we should make default SED intervals very long in examples, because it's pretty cumbersome to test such a device.

Reply:
Then I think I should preserve this change. Since a sleepy device with 2s interval is even more "active" than network jitters sometimes, and it will pass the tests without this PR.

While this PR is to resolve the issues of more sleepy devices (i.e. the devices that wake up every tens of seconds)